tests/DataFixtures/ORM/NetworksFixture.php

```diff
@@ -6,12 +6,20 @@
 use BBC\ProgrammesPagesService\Data\ProgrammesDb\Entity\Network;
 use BBC\ProgrammesPagesService\Data\ProgrammesDb\Entity\Service;
 use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
 use Doctrine\Common\Persistence\ObjectManager;
 
-class NetworksFixture extends AbstractFixture
+class NetworksFixture extends AbstractFixture implements DependentFixtureInterface
 {
     private $manager;
 
+    public function getDependencies()
+    {
+        return [
+            __NAMESPACE__ . '\\ImagesFixture',
```

Comment:
After doing all this the first time around, I realised `ImagesFixture::class` is a lot simpler